pkg/commands/access/actions/list.go

```diff
@@ -39,5 +54,49 @@ func AccessList() *cobra.Command {
 		},
 	}
 
+	flags := cmd.Flags()
+	flags.StringVarP(&integrationFilter, integrationFilterFlagName, "i", "", "The integration id or type/name, for example: \"aws-account/My AWS integration\"")
+	flags.StringVarP(&bundleFilter, bundleFilterFlagName, "b", "", "filter by bundle name or id")
```

Comment:
do we really want bundle id? I mean will the users ever use that?

Reply:
thats what we talked about and its in the PRD

Reply:
👍